NewHorizons/Components/EyeOfTheUniverse/InstrumentZone.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace NewHorizons.Components.EyeOfTheUniverse
+{
+    public class InstrumentZone : MonoBehaviour
```

Comment:
Why does this exist? Probably for a good reason so can you comment in the class why

Reply:
(I ask bc its an empty class)